runtime/ops/os/mod.rs

```diff
@@ -341,6 +343,120 @@ fn op_uid(
   Ok(None)
 }
 
+#[derive(Serialize)]
+struct CpuUsage {
+  system: usize,
+  user: usize,
+}
+
+#[op2]
+#[serde]
+fn op_runtime_cpu_usage() -> CpuUsage {
```

Comment:
TIPS: returning a tuple is better performance instead of a struct.

https://github.com/denoland/deno_core/blob/0d64502ac3562229873cc0a36261afb0bdc8e372/core/convert.rs#L55-L66

Reply:
But runtime_memory_usage also returns a struct.
I just followed the memory_usage op.

Reply:
> TIPS: returning a tuple is better performance instead of a struct.
> 
> https://github.com/denoland/deno_core/blob/0d64502ac3562229873cc0a36261afb0bdc8e372/core/convert.rs#L55-L66

Should I create two functions in deno_ns or put all logic in a lambda for converting?

https://github.com/denoland/deno/blob/4ded2696019c38725d1cffcf89a3bd1afabda812/runtime/js/90_deno_ns.js#L58-L59